src/app/ApplicationTemplate.Mobile/Android/Resources/values/Styles.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<resources>
+	<style name="AppTheme" parent="Theme.AppCompat.Light">
+
+		<!-- This removes the ActionBar -->
+		<item name="windowActionBar">false</item>
+		<item name="android:windowActionBar">false</item>
+		<item name="windowNoTitle">true</item>
+		<item name="android:windowNoTitle">true</item>
+
+	</style>
+	<style name="Theme.AppCompat.Translucent">
+		<item name="android:windowIsTranslucent">true</item>
+		<item name="android:windowAnimationStyle">@android:style/Animation</item>
+	</style>
```

Comment:
I'm wondering if it should have the same content as here https://github.com/nventive/UnoApplicationTemplate/blob/main/src/app/ApplicationTemplate.Droid/Resources/values/Styles.xml

Reply:
May I could had [this](https://github.com/dotnet/runtime/issues/74021)?